views/move/modal_move.php

```diff
@@ -9,26 +9,29 @@
 /* @var $model MoveForm */
 ?>
 
-<?php ModalDialog::begin(['header' => Yii::t('CfilesModule.base', '<strong>Move</strong> files'), 'size' => 'small']) ?>
+<?php ModalDialog::begin(['header' => Yii::t('CfilesModule.base', '<strong>Move</strong> files'), 'size' => 'small']); ?>
 
     <?php $form = ActiveForm::begin(); ?>
         <div class="modal-body">
-            <br />
+            <br>
 
-            <?= $this->render('directory_tree', ['root' => $model->root]) ?>
+            <?= $this->render('directory_tree', ['root' => $model->root]); ?>
 
-            <?= $form->field($model, 'destId')->hiddenInput(['id' => 'input-hidden-selectedFolder'])->label(false) ?>
+            <?= $form->field($model, 'destId')->hiddenInput(['id' => 'input-hidden-selectedFolder'])->label(false); ?>
 
-            <?php foreach ($model->selection as $index => $item) : ?>
-                <input class='input-hidden-selectedItem' type='hidden' name='selection[]' value='<?= $item ?>' />
-            <?php endforeach; ?>
+            <?php if (is_array($selectedItems)) {
+                foreach ($selectedItems as $index => $item) {
```

Comment:
Where did you get the `$selectedItems` from? [This](https://github.com/humhub/humhub-modules-cfiles/blob/master/controllers/MoveController.php#L46) is what is given to the view. Seems like you are using a rather old branch.